birkof/netopia-mobilpay-bundle/1.0/config/packages/netopia_mobilpay.yaml

```diff
@@ -0,0 +1,7 @@
+netopia_mobilpay:
+    payment_url:        '%env(NETOPIA_MOBILPAY_PAYMENT_URL)%'
+    public_cert:        '%env(NETOPIA_MOBILPAY_PUBLIC_CERT)%'
+    private_key:        '%env(NETOPIA_MOBILPAY_PRIVATE_KEY)%'
+    signature:          '%env(NETOPIA_MOBILPAY_SIGNATURE)%'
+    return_url:         'netopia_mobilpay_return_url'
```

Comment:
Shouldnt these routes be registered?

Reply:
Okey, Maybe create a routing file with those defined? (But just pointing to `/`)?

Reply:
Nice hint! Lemme switch to...

Reply:
@Nyholm changes pushed 👍